WordPress/Classes/ViewRelated/Reader/ReaderDetailViewController.swift

```diff
@@ -738,10 +738,17 @@ public class ReaderDetailViewController : UIViewController, UIViewControllerRest
         let isOfflineView = ReachabilityUtils.isInternetReachable() ? "no" : "yes"
         let detailType = post!.topic?.type == ReaderSiteTopic.TopicType ? DetailAnalyticsConstants.TypePreviewSite : DetailAnalyticsConstants.TypeNormal
 
+
         var properties = ReaderHelpers.statsPropertiesForPost(post!, andValue: nil, forKey: nil)
         properties[DetailAnalyticsConstants.TypeKey] = detailType
         properties[DetailAnalyticsConstants.OfflineKey] = isOfflineView
-        WPAppAnalytics.track(WPAnalyticsStat.ReaderArticleOpened, withProperties: properties)
+        WPAppAnalytics.track(.ReaderArticleOpened, withProperties: properties)
```

Comment:
Is `.ReaderArticleOpened` supposed to fire here with both `WPAppAnalytics.track(.ReaderArticleOpened, withProperties: properties)` as well as below with `WPAppAnalytics.trackTrainTracksInteraction(.ReaderArticleOpened, withProperties: railcar)`? Just checking, not super familiar with what's going on here.